plugins/performance-lab/includes/admin/load.php

```diff
@@ -471,3 +471,28 @@ function perflab_get_plugin_settings_url( string $plugin_slug ): ?string {
 
 	return null;
 }
+
+/**
+ * Prints the Performance Lab install notice after each feature plugin's row meta.
+ *
+ * @since n.e.x.t
+ *
+ * @param string $plugin_file Plugin file.
+ */
+function perflab_print_row_meta_install_notice( string $plugin_file ): void {
+	if ( ! in_array( strtok( $plugin_file, '/' ), perflab_get_standalone_plugins(), true ) ) {
+		return;
+	}
+
+	$message = sprintf(
+		/* translators: %s: link to Performance Lab settings screen */
+		__( 'This plugin is installed by <a href="%s">Performance Lab</a>', 'performance-lab' ),
+		esc_url( add_query_arg( 'page', PERFLAB_SCREEN, admin_url( 'options-general.php' ) ) )
+	);
+
+	printf(
+		'<div class="requires"><p>%1$s</p></div>',
```

Comment:
```suggestion
		'<div class="requires">%s</div>',
```

The `<p>` tag added bottom margin that is not needed.

Reply:
@mukeshpanchal27 That's true, but core is [using](https://github.com/WordPress/wordpress-develop/blob/27151ba0b9693f95717785c164e2f30e64a92627/src/wp-admin/includes/class-wp-plugins-list-table.php#L1596-L1600) a paragraph in such cases:

![image](https://github.com/WordPress/performance/assets/134745/47f9e3b8-9a96-41c5-a37c-3919923d8f80)

So it seems best to use it here for consistency.